files/image_config/topology/topology.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+start () {
```

Comment:
this topology is very specific to a hwsku, should be move to a hardware sku directory.

Reply:
Done